test/unit/test_sap_cli_srvb.py

```
@@ -0,0 +1,372 @@
+#!/usr/bin/env python3
+'''CLI tests for sapcli srvb (Service Binding).'''
+
+# pylint: disable=missing-function-docstring
+
+import unittest
+from unittest.mock import call, patch, Mock
```

Flake8 (7.3.0)

[error] 'unittest.mock.call' imported but unused

(F401)